lua/autorun/MakeSphericalCore.lua

```diff
@@ -137,6 +137,7 @@ if SERVER then
 		
 		data.radius = radius
 		ent.noradius = data.noradius
+		PrintTable(data)
```

Comment:
This looks like some debug printing, is it OK to remove this before I merge?

Reply:
Yes, forgot about this lol